dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -44,16 +44,19 @@
 DEFAULT_BACKEND = ALLOWED_BACKENDS[0]
 ALL_INTEGRATIONS = [
     "cassandra",
+    "celery",
     "kerberos",
     "mongo",
     "pinot",
-    "celery",
     "trino",
 ]
-ALLOWED_INTEGRATIONS = [
-    *ALL_INTEGRATIONS,
-    "all",
-]
+ALLOWED_INTEGRATIONS = sorted(
```

Comment:
Using `sorted` here so the output on the CLI help screen will be sorted.  Not necessarily required, but I liked it.

Reply:
Good.